modify_team_data.rb

```diff
@@ -0,0 +1,9 @@
+require 'yaml'
+
+employee_data = YAML.load_file('C:\dev_projekte_git\Helmcraft_GmbH\devcraft_landingpage\landingpage\_data\teammembers.yml')

```

Comment:
Das wird aber nur auf deinem Computer funktionieren. 😉 

Generell verwende in so einem Fall immer Pfade, die relativ zum Projektverzeichnis sind. Wir brauchen diese Ruby-Datei aber gar nicht, drum bitte löschen.